pkgs/development/coq-modules/smtcoq/default.nix

```
@@ -0,0 +1,26 @@
+{ lib, stdenv, mkCoqDerivation, coq, trakt, cvc4, veriT, version ? null }:
+with lib;
+
+mkCoqDerivation {
+  pname = "smtcoq";
+  owner = "smtcoq";
+
+  release."itp22".rev    = "1d60d37558d85a4bfd794220ec48849982bdc979";
+  release."itp22".sha256 = "sha256-CdPfgDfeJy8Q6ZlQeVCSR/x8ZlJ2kSEF6F5UnAespnQ=";
```

```suggestion
  release."itp22" = {
    rev    = "1d60d37558d85a4bfd794220ec48849982bdc979";
    sha256 = "sha256-CdPfgDfeJy8Q6ZlQeVCSR/x8ZlJ2kSEF6F5UnAespnQ=";
  };
```
If `"itp22"` changes it won't have to be updated twice.